src/components/Article-buttons.jsx

```diff
@@ -0,0 +1,106 @@
+import Button from "@mui/material/Button";
+import { IconButton } from "@mui/material";
+import ButtonGroup from "@mui/material/ButtonGroup";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import ArrowCircleDownOutlinedIcon from "@mui/icons-material/ArrowCircleDownOutlined";
+import ArrowCircleUpOutlinedIcon from "@mui/icons-material/ArrowCircleUpOutlined";
+import { updateArticleVotes } from "../utils/api";
+import { useState } from "react";
+
+const ArticleButtons = ({ article, openComments, showComments }) => {
+  const [votes, setVotes] = useState(article.votes);
+  const [upvote, setUpvote] = useState(false);
+  const [downvote, setDownvote] = useState(false);
+  const [upvoteCol, setUpvoteCol] = useState("primary");
```

Comment:
these variable names are unclear to me what they are, and I'm not sure what primary means